### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12624,6 +12626,8 @@ enforcable->enforceable
 enforceing->enforcing
 enforcmement->enforcement
 enforcment->enforcement
+enfore->enforce
+enfores->enforces
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:12630]:
```text
enfored->enforced
```
https://github.com/search?q=enfored&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -15658,6 +15662,18 @@ funtionally->functionally
 funtionalty->functionality
 funtioning->functioning
 funtions->functions
+funvtion->function
+funvtional->functional
+funvtionality->functionality
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:15667]:
```text
funvtionalities->functionalities
```
https://github.com/search?q=funvtionalities&type=code

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -15658,6 +15662,18 @@ funtionally->functionally
...
+funvtioned->functioned
+funvtioning->functioning
+funvtions->functions
+funxtion->function
+funxtional->functional
+funxtionality->functionality
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:15673]:
```text
funxtionalities->functionalities
```
https://github.com/search?q=funxtionalities&type=code

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -19064,6 +19081,11 @@ itnerest->interest
 itnerface->interface
 itnerfaces->interfaces
 itnernal->internal
+itnerprelation->interpretation
+itnerpret->interpret
+itnerpretation->interpretation
+itnerpretaton->interpretation
+itnerpreter->interpreter
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:19088]:
```text
itnerpreted->interpreted
itnerprets->interprets
itnerpreting->interpreting
```
https://github.com/search?q=itnerpreted&type=code
https://github.com/search?q=itnerprets&type=code
https://github.com/search?q=itnerpreting&type=code

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -23505,6 +23527,8 @@ partiton->partition
 partitoned->partitioned
 partitoning->partitioning
 partitons->partitions
+partiula->particular
+partiular->particular
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:23531]:
```text
partiulars->particulars
partiularly->particularly
```
https://github.com/search?q=partiulars&type=code
https://github.com/search?q=partiularly&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:23537]: [47c97b6](https://github.com/codespell-project/codespell/pull/2427/commits/47c97b6fb43b0e0a90126552064985a7d2bad7a6)

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:19092]: [47c97b6](https://github.com/codespell-project/codespell/pull/2427/commits/47c97b6fb43b0e0a90126552064985a7d2bad7a6)

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:15676]: [47c97b6](https://github.com/codespell-project/codespell/pull/2427/commits/47c97b6fb43b0e0a90126552064985a7d2bad7a6)

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:15669]: [47c97b6](https://github.com/codespell-project/codespell/pull/2427/commits/47c97b6fb43b0e0a90126552064985a7d2bad7a6)

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:12631]: [47c97b6](https://github.com/codespell-project/codespell/pull/2427/commits/47c97b6fb43b0e0a90126552064985a7d2bad7a6)

**Pull request merged**